### DISCUSSION TITLE
Banner edit by using tarteaucitron.io dashboard

### DISCUSSION THREAD

**Discussion by QuentinV33** [AmauriC/tarteaucitron.js#803] [Q&A] [answered]
Hello,

I'm trying to edit my banner by using the "tarteaucitron" dashboard but I'm kinda stuck for some CSS requests.
Some color changes or Border radius work very well (using CSS). But I'm trying to add a solid border and other stuffs on these CSS selector and it doesn't work.
Here is my CSS :

```
#tarteaucitronAllDenied2 {
border: solid 2px;
border-radius: 50px;
padding: 10px;
border-color: #50748a;
}
```

Does my problem comes from my selector or that's juste something that I can't change from the dashboard ?

Thank's for your help !

**Comment by AmauriC** [comment, marked as answer]
Hey,

Use this one to overwrite the previous rules:
```
#tarteaucitronRoot #tarteaucitronAllDenied2 {
   border: solid 2px;
   border-radius: 50px;
   padding: 10px;
   border-color: #50748a;
}
```

**Reply by QuentinV33** [reply]
My bad !

Thank's a lot ! 😁

**Reply by QuentinV33** [reply]
By the way, do you know how could I switch the Cookie emoji with a logo or something like that ?

**Reply by AmauriC** [reply]
```
html body #tarteaucitronRoot div#tarteaucitronAlertBig:before {
    background: url(https://tarteaucitron.io/img/logo.png) center no-repeat;
    width: 100%;
    height: 50px;
    display: block;
    content: ' ';
}
```

😎

**Reply by QuentinV33** [reply]
Thank's a lot ! 😄
